tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -667,12 +667,22 @@ def _run_omb(self, scale: ScaleParameters):
         bench_node = self.preallocated_nodes[0]
         worker_nodes = self.preallocated_nodes[1:]
 
+        # TODO: remove these overrides once the cause of latency
```

Comment:
If you link the issue in the comment, it'll help future readers get context

Reply:
Should I add a link or the issue number?

Reply:
Just a link to the issue in the comment? # Tracking issue: https://github.com/redpanda-data/redpanda/issues/xxxx

Reply:
Yep, we use URLs because issue numbers are ambiguous as to what project they come from (and if we ever switch issue tracker in the future)